.editorconfig

```diff
@@ -0,0 +1,8 @@
+# EditorConfig is awesome: https://EditorConfig.org
+root = true
+
+[*]
```

Comment:
```suggestion
[*.py]
```

Reply:
This wouldn't have helped with #1560 as requirements.txt wouldn't match *.py and thus my editor would have made tabs there. How about 49f9cdc1528134d5734537701c54d0898fb2b602 instead?